src/Forms/Layout/Row.php

```diff
@@ -70,10 +70,23 @@ public function __call($function, $args)
                 $element->setRow($this);
             }
         } catch (\ReflectionException $e) {
-            $element = $this->factory->createContent(sprintf('Cannot %1$s. This form element does not exist in the current FormFactory', $function).': '.$e->getMessage());
+            $element = $this->factory->createContent(__('Cannot {function}. This form element does not exist in the current FormFactory: ', [
+                'function' => $function,
+                'message' => $e->getMessage(),
+            ]));
         } catch (\Exception $e) {
-            $element = $this->factory->createContent(sprintf('Cannot %1$s. Error creating form element.', $function).': '.$e->getMessage());
+            $element = $this->factory->createContent(__('Cannot {function}. Error creating form element: {message}', [
+                'function' => $function,
+                'message' => $e->getMessage(),
+            ]));
         } finally {
+            if (!($element instanceof OutputableInterface)) {
+                if (($element_type = gettype($element)) === 'object') $element_type = get_class($element);
+                $element = $this->factory->createContent(__('{function} returned {type} instead of an outputable form element.', [
+                    'type' => $element_type,
+                    'function' => $function,
+                ]));
```

Comment:
```suggestion
                $element = $this->factory->createContent(strtr('{function} returned {type} instead of an outputable form element.', [
                    '{type}' => $element_type,
                    '{function}' => $function,
                ]));
```